cmd/modelcmd/controller.go

```diff
@@ -418,8 +407,7 @@ type OptionalControllerCommand struct {
 // SetFlags initializes the flags supported by the command.
 func (c *OptionalControllerCommand) SetFlags(f *gnuflag.FlagSet) {
 	c.CommandBase.SetFlags(f)
-	f.BoolVar(&c.ClientOnly, "client-only", false, "Client operation only; controller not affected")
-	f.BoolVar(&c.ControllerOnly, "controller-only", false, "Controller operation only; client not affected")
+	f.BoolVar(&c.Client, "client", false, "Command to operate on or with a copy from this client")
```

Comment:
i don't parse the description

Reply:
Changed to "Client operation" echoing deprecated "Local operation only,..." and "Controller to operate in"